roles/common/files/libexec/smart.sh

```diff
@@ -103,6 +110,17 @@ preflight ()
     echo "yum/apt-get install smartmontools"
     exit 3
   fi
+
+  # Check for nvme devices and nvme-cli executable
+  if cat /proc/partitions | grep -q nvme
+  then
+    nvme=true
+    if ! [ -x "$nvmecli" ]
+    then
+      echo "ERROR - NVMe Device detected but no nvme-cli executable"
+      exit 3
```

Comment:
Hm good question.  I think the reasoning I had for it exiting was to make sure it alerted, and thus got looked at and fixed.

Otherwise, if spinners get checked and are healthy, it'll stay green in nagios but won't indicate any problem with the executable being missing.